content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers.rst

```diff
@@ -1,147 +1,230 @@
-========================
-Work with serial numbers
-========================
+====================================
+Use serial numbers to track products
+====================================
 
-To track products with serial numbers, you can use the serial number
-tracking. With them, you can track the current location of the product
-and, when the products are moved from one location to another, the
-system will automatically identify the current location of the product,
-based on its last movement.
+*Serial Numbers* are one of the two ways to identify and track products in Odoo. A serial number
+is a unique identifier assigned incrementally (or sequentially) to an item or product, so as to
+identify it in order to distinguish it from other items and products.
 
-If you want to do it or run very advanced traceability, the Odoo
-double-entry management is the feature you need.
+Serial numbers can consist of many different types of characters: they can be strictly numerical;
+they can contain letters and other typographical symbols; and they might even consist entirely of
+character string. The goal of assigning serial numbers to individual products is to make sure that
```

Comment:
> they might even consist entirely of character string.

I don't know what this is supposed to mean.

Reply:
Haha